app/scripts/collections/forms.coffee

```diff
@@ -205,9 +205,8 @@ define [
     ############################################################################
 
     getFetchAllFieldDBFormsURL: ->
-      url = globals.applicationSettings.get 'baseDBURL'
-      pouchname = globals.applicationSettings.get 'activeFieldDBCorpus'
-      "#{url}/#{pouchname}/_design/pages/_view/datums_chronological"
+      url = globals.applicationSettings.get('fieldDBApplication').corpus.url
```

Comment:
The `url` you get here: it has the pouchname in it? I.e., it's <url>/<pouchname>/ ? Just checking ...